### PR TITLE
feat(server): Return the `Host` or `X-Forwarded-Host` header field to a hostname.

### PR DESCRIPTION
**Pull request by getspooky**

### CODE REVIEW

File: server/api.ts [hunk 1]
```diff
@@ -43,6 +43,10 @@ export class Request implements APIRequest {
     return this.#req.method
   }
 
+  get host(): string {
```
Comment (ije) [inline server/api.ts:46]: maybe it should be `hostname` that removes the port?

Comment (getspooky) [reply server/api.ts:46]: yes I will rename it to `hostname`

Comment (ije) [reply server/api.ts]: nice! thanks

Comment (getspooky) [reply server/api.ts]: @ije fixed

Comment (ije) [reply server/api.ts]: Screenshot 2021-04-14 at 00 49 24

@getspooky  we should strip the **port** as `hostname`.

Comment (getspooky) [reply server/api.ts]: @ije we can simply use `(this.#req.conn.remoteAddr as Deno.NetAddr).hostname`

Comment (ije) [reply server/api.ts]: great!

**Comment by ije** [comment]
@getspooky thanks for the new feature!

**Pull request merged**